Classes/Domain/Service/StyleAndJavascriptInclusionService.php

```diff
@@ -61,7 +61,7 @@ class StyleAndJavascriptInclusionService
     public function getHeadScripts(): string
     {
         return $this->build($this->javascriptResources, function ($uri, $additionalAttributes) {
-            return '<script src="' . $uri . '" ' . $additionalAttributes . '></script>';
+            return '<script src="' . $uri . '" ' . $additionalAttributes . ' defer></script>';
```

Comment:
if one configures 

```yaml
resource: fooo
defer: true
```

or
```yaml
resource: fooo
attributes:
  defer: true
```

there will be actually a second `defer` attribute in $additionalAttributes already 🤯 shouldn't be a real problem though ^^

i could imagine omitting the `defer` below in the `htmlAttributesArrayToString`.

And it seems that if someone uses an esm plugin with 

```yaml
resource: fooo
attributes:
  type: "module"
```

we can omit this defer tag, as its implicitly set and doppeltgemoppelt? https://wiki.selfhtml.org/wiki/HTML/Attribute/defer

Reply:
But i could confirm that `<script src="plugin" type="module" defer=""></script>` works ;)

Reply:
And i see you fixed my initial concerns via https://github.com/neos/neos-ui/pull/3682/commits/4325fb0d5c239954e37491cf56c683ad4a85dbfd ;)